src/gui/spatialArrow.js

```diff
@@ -0,0 +1,324 @@
+createNameSpace("realityEditor.gui.spatialArrow");
+
+import * as THREE from '../../thirdPartyCode/three/three.module.js';
+
+(function (exports) {
+    
+    let canvasContainer;
+    let canvas;
+    let ctx;
+    let screenW, screenH;
+    let screenRatio;
+    
+    let color, colorLighter;
```

Comment:
If you run `npm test` or `npm run lint`, you'll see a list of automatically detected code linting errors (you may need to install eslint first) – such as removing variables that are defined but not used. GitHub also highlights these issues when you make the pull request. Can you go through and correct each of these linting errors?

Reply:
Thx for the tip! I executed npm run lint and removed some unused code. Thx Ben!